cfme/fixtures/ansible_tower.py

```diff
@@ -0,0 +1,121 @@
+                                "options": {
+                                    "protected": False
+                                },
+                                "label": "Limit",
+                                "position": 1,
+                                "reconfigurable": True,
+                                "visible": True,
+                                "type": "DialogFieldTextBox",
+                                "resource_action": {
+                                    "resource_type": "DialogField"
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "description": "Basic 2",
+                        "display": "edit",
+                        "label": "Survey",
+                        "position": 1,
+                        "dialog_fields": [
+                            {
+                                "name": "param_Department",
+                                "description": "",
+                                "display": "edit",
+                                "required": True,
+                                "default_value": "QE",
+                                "values": [
+                                    [
+                                        "HR",
+                                        "HR"
+                                    ],
+                                    [
+                                        "PM",
+                                        "PM"
+                                    ],
+                                    [
+                                        "QE",
+                                        "QE"
+                                    ]
+                                ],
+                                "options": {},
+                                "label": "Survey",
+                                "position": 0,
+                                "reconfigurable": True,
+                                "visible": True,
+                                "type": "DialogFieldDropDownList",
+                                "resource_action": {
+                                    "resource_type": "DialogField"
+                                }
+                            }
+                        ]
+                    }
+                ]
+            }
+        ]
+    }
+
+    service_dialog = _creating_skeleton(request, appliance, "service_dialogs",
+        [data])
+    return service_dialog[0]
+
+
+@pytest.fixture(scope="function")
+def ansible_tower_dialog(request, appliance):
+    """Returns service dialog object."""
+    rest_resource = ansible_tower_dialog_rest(request, appliance)
+    service_dialogs = appliance.collections.service_dialogs
+    service_dialog = service_dialogs.instantiate(
+        label=rest_resource.label,
+        description=rest_resource.description)
+    return service_dialog
```

Comment:
Can you use yield intead and Clean up dialog after yield.

Reply:
Done